Makefile

```diff
@@ -30,6 +30,8 @@ ifeq "$(TAG_SQLITE3)" ""
 	exit 1
 endif
 
+	cd lxd/config/generate && CGO_ENABLED=0 go build -o $(GOPATH)/bin/lxd-doc
```

Comment:
Can we use the same sort of invocation we do for the other commands, e.g.

```
CGO_ENABLED=0 go install -v -tags netgo ./lxd-migrate
```

Reply:
@gabrielmougard Can you update this?